packages/core/src/dialects/ibmi/data-types.ts

```diff
@@ -8,16 +8,27 @@ export class UUID extends BaseTypes.UUID {
 }
 
 export class BOOLEAN extends BaseTypes.BOOLEAN {
+  protected supportsNativeBooleans() {
+    const databaseVersion
+      = Number.parseFloat(this._getDialect().sequelize.options.databaseVersion || this._getDialect().defaultVersion);
+
+    return databaseVersion >= 7.5;
```

Comment:
We can use the `gt` function from [semver](https://www.npmjs.com/package/semver) to compare this to getDatabaseVersion()